packages/headless/src/api/commerce/product-listings/product-listing-api-client.ts

```diff
@@ -15,6 +15,28 @@ import {
   ProductListingRequest,
   ProductListingSuccessResponse,
 } from './product-listing-request';
+// eslint-disable-next-line node/no-extraneous-import
+import {ApolloClient, InMemoryCache, gql} from '@Apollo/client';
+
+const QUERY = gql`
+  {
+    placementContent(
+      mode: PREVIEW
+      previewOptions: {campaignId: "foo"}
+      placementId: "foo"
+      attributes: {visitor: {id: ""}}
+    ) {
+      content
+      visitorId
+      callbackData
+    }
+  }
+`;
```

Comment:
For this test I used the query Andrey sent in the channel. Obviously in a real implementation this wouldn't here and the values would not be hardcoded.